domains/arsdorint.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "nambp2003",
+        "email": "e0851451@u.nus.edu"
+      },
+      "record": {
+        "URL": "arsdorint"
```

Comment:
Please specify a valid URL

Reply:
Ok sorry, I forget to check the document page. I would change the URL to 
"TXT": "arsdorint". Is this ok?

Reply:
> Ok sorry, I forget to check the document page. I would change the URL to "TXT": "arsdorint". Is this ok?

No since it isn't actually doing anything

Reply:
OK nevermind I just add an URL to it, "https://github.com/nambp2003"